libs/source/Source.mli

```diff
@@ -0,0 +1,60 @@
+(* Cooper Pierce
+ *
+ * Copyright (c) Semgrep Inc.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * version 2.1 as published by the Free Software Foundation, with the
+ * special exception on linking described in file LICENSE.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the file
+ * LICENSE for more details.
+ *)
+
+(** Sources of content which may be consumed by Semgrep.
+
+   A source is the {e user-relevant location} where content (e.g., source code)
```

Comment:
Please do not use location; this term is heavily charged in semgrep to mean a source code region/range.

Reply:
Reworded in 5fee815255b446b1e3360b63d94021332adf6f60